operators/operator_sdk/osdk-generating-csvs.adoc

```diff
@@ -37,6 +37,11 @@ include::modules/olm-enabling-operator-for-multi-arch.adoc[leveloffset=+1]
 include::modules/olm-arch-os-support.adoc[leveloffset=+2]
 
 include::modules/osdk-suggested-namespace.adoc[leveloffset=+1]
+include::modules/osdk-operatorconditions.adoc[leveloffset=+1]
+.Additional resources
+
+* xref:../../operators/understanding/olm/olm-understanding-olm.adoc#olm-understanding-olm[OLM concepts -> Operator conditions]
```

Comment:
Will update this link after https://github.com/openshift/openshift-docs/pull/28538 merges.